docs/browsers-support/README.md

```diff
@@ -24,11 +24,31 @@ We use [Babel](https://babeljs.io/) with [@babel/preset-env](https://babeljs.io/
 
 Language can be transpiled but features must be polyfilled.
 
-Install [@babel/polyfill](https://babeljs.io/docs/en/babel-polyfill) and set `@babel/preset-env` option `useBuiltIns` to `usage` to automatically load the necessary polyfills based on your code and Browserslist settings (see above).
+Install [core-js@3](https://github.com/zloirock/core-js), set `@babel/preset-env` option `useBuiltIns` to `usage` to automatically load the necessary polyfills based on your code and Browserslist settings (see above) and set the `corejs` version to `3`.
```

Comment:
It's a bit long and mixing various concepts in the same sentence, can you add some punctuation so we understand it better?

Reply:
Is it better now ?